testing/src/cypress/e2e/02-institution/06-manage-users.cy.ts

```diff
@@ -56,7 +57,7 @@ describe("Manage Users", () => {
   });
 });
 
-describe("Manage Users - Add new user", () => {
+describe.skip("Manage Users - Add new user", () => {
```

Comment:
There could be a comment or to-do for these skips.

Reply:
updated